crates/cxx-qt-gen/src/writer/rust/mod.rs

```diff
@@ -41,9 +41,9 @@ fn cxx_bridge_common_blocks(qobject: &GeneratedRustQObject) -> Vec<TokenStream>
                 /// <https://github.com/dtolnay/cxx/issues/683>
                 type #cxx_qt_thread_ident;
 
-                /// Retrieve an immutable reference to the Rust struct backing this C++ object
                 #[cxx_name = "unsafeRust"]
-                fn rust(self: &#cpp_struct_ident) -> &#rust_struct_ident;
+                #[doc(hidden)]
+                fn ffi_rust(self: &#cpp_struct_ident) -> &#rust_struct_ident;
```

Comment:
do we use `ffi_` as our common "hidden" method naming ? As we have the same problem with `qt_thread()` ? or should we use like `cxx_qt_` or `cxx_qt_ffi_` ?

Namespacing to our things to at least `cxx_qt` would avoid any collisions accidentally

Reply:
We could do `ccx_qt_ffi` :thinking: 
Let me check if that also fixes our "similarity" issues.

Reply:
![image](https://github.com/KDAB/cxx-qt/assets/84974957/339be211-43b2-454a-82cd-8a483d93f21a)
At least for `rust()` and `rust_mut()`, you no longer get suggested the `ffi_rust` stuff.

@ahayzen-kdab Can you check that `cxx_qt_ffi` also works for `qt_thread`?
As that may have more overlap with the longer method name.

Reply:
Yup it makes it different enough too

```
build] error[E0599]: no method named `qt_thread` found for struct `Pin<&mut threading::ffi::ThreadingWebsiteQt>` in the current scope
[build]   --> examples/qml_features/rust/src/threading.rs:90:34
[build]    |
[build] 90 |             let qt_thread = self.qt_thread();
[build]    |                                  ^^^^^^^^^ method not found in `Pin<&mut ThreadingWebsiteQt>`
[build]    |
[build]    = help: items from traits can only be used if the trait is in scope
[build] help: the following trait is implemented but not in scope; perhaps add a `use` for it:
[build]    |
[build] 11 | use cxx_qt::Threading;
[build]    |
```